critiquebrainz/frontend/templates/artist.html

```diff
@@ -10,10 +10,26 @@
          {% if member.begin %}title="{{ _('Member since %(date)s', date=member.begin) }}"{% endif %}>
         {{ member.artist.name }}
       </a>
-      {% if member.end %}
-        <span class="text-muted">({{ member.begin }} - {{ member.end }})</span>
-      {% elif member.artist.disambiguation %}
-        <span class="text-muted">({{ member.artist.disambiguation }})</span>
+      {% if member.periods -%}
+        <span class="text-muted">(
+        {%- for period in member.periods -%}
+            {{ period[0] + '-' + period[1] }}
+            {%- if not loop.last -%}
+              {{ ', ' }}
+            {%- endif -%}
+        {%- endfor -%}
+        )</span>
+      {% endif %}
+      {% if member['attribute-list'] and not member.ended %}
```

Comment:
Warning probably because variable names can't have `-` character in them. So it's probably better to use the first option. I know some of existing code is inconsistent either, but there's no reason to make it worse.